src/modality-specific-files/magnetic-resonance-imaging-data.md

```diff
@@ -1,5 +1,25 @@
 # Magnetic Resonance Imaging
 
+## Data format
+
+All MRI imaging data MUST be stored using the
+[NIfTI-1 or NIFTI-2](https://nifti.nimh.nih.gov/) file format (`.nii`).
+The ANALYZE-7.5 file format (`.hdr`/`.img`), despite being very similar to NIFTI-1,
+is not allowed.
```

Comment:
Wondering if it makes sense to mention things we do not allow: the list of those is possibly infinite.

Reply:
Actually this is the my reason of doing this merge request. Based on Nifty specs, hdr/img are also considered as Nifty, so technically accepted by BIDS. 

I think this precision is important, as it avoids a possible confusion. But if you wish to replace `ANALYZE-7.5` by `two-file NIFTI` or `header-image NIFTI`, I will not protest.

Reply:
ah I see. OK ignore this comment then.